core/meta/inc/TClass.h

```diff
@@ -232,9 +235,9 @@ friend class TProtoClass;
 
    typedef void (*StreamerImpl_t)(const TClass* pThis, void *obj, TBuffer &b, const TClass *onfile_class);
 #ifdef R__NO_ATOMIC_FUNCTION_POINTER
-   mutable StreamerImpl_t fStreamerImpl;  //! Pointer to the function implementing the right streaming behavior for the class represented by this object.
+   mutable StreamerImpl_t fStreamerImpl; //! Pointer to the function implementing streaming for this class
```

Comment:
The original comment is intentionally pendantic 'this class' could be meaning 'the class named TClass' or the "the class represented by the instance of TClass"